openkfw/TruBudget
# 1955 update workflow

### Checklist




### Description


### How to test

1. Edit a workflowitem
2. Add a file, or add a link, or both
3. If file is added, check that V2 endpoint is used
4. Check that all added files and links are correctly added




Closes #1955
